### PR TITLE
WMS ID#11441 

### PR DESCRIPTION
**Pull request by gautamm2707**
# Add here your ID#11441 

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ ] All filenames are lower case (including folders, images, files, etc.)
- [ ] Filenames are descriptive
- [ ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ ] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by aberinnj** [comment]
Hi,

Please make sure the filenames (the images) are **descriptive**

Thank you

File: linux-pam-sssd-identity-domains/add-sso-policy/images/image1.png
Comment (aberinnj) [inline linux-pam-sssd-identity-domains/add-sso-policy/images/image1.png:1]: Please make sure the filenames are descriptive as mentioned in the self-QA doc

**Review by aberinnj — CHANGES_REQUESTED** [review]

File: linux-pam-sssd-identity-domains/add-sso-policy/images/image1.png
Comment (gautamm2707) [reply linux-pam-sssd-identity-domains/add-sso-policy/images/image1.png:1]: made changes in each image's description. kindly validate and confirm

Comment (gautamm2707) [reply linux-pam-sssd-identity-domains/add-sso-policy/images/image1.png:1]: Note: changes have been made in the description of the images (visible once you hover them on the screen) in the md files. 
No changes made on the "name" of the image files, as that does not makes sense; Kindly validate and confirm

Comment (aberinnj) [reply linux-pam-sssd-identity-domains/add-sso-policy/images/image1.png:1]: Hi, the names of the images are exactly what I was referring to, it is in the self-QA doc

Comment (gautamm2707) [reply linux-pam-sssd-identity-domains/add-sso-policy/images/image1.png:1]: Hi ,the image names have been updated with more descriptive names. Kindly validate and merge.

**Review by aberinnj — APPROVED** [review]

**Review by aberinnj — APPROVED** [review]

**Comment by aberinnj** [comment]
The rest lgtm!

I noticed that the workshop outline on WMS are not updated, please update it and I'll merge

**Pull request merged**